clientv3/op.go

```diff
@@ -415,6 +424,22 @@ func WithCreatedNotify() OpOption {
 	}
 }
 
+// WithFragmentedResponse makes the watch server send watch responses
+// that are too large to send over the rpc stream in fragments.
+// The fragmenting feature is an opt-in feature in order to maintain
+// compatibility with older versions of clients.
+func WithFragmentedResponse() OpOption {
+	return func(op *Op) { op.fragmentResponse = true }
+}
+
+// WithFragments allows watch clients to passively relay their receieved
+// fragmented watch responses.
+// This option is handy for the watch client's belonging to watch proxies,
+// because these watch clients can be set to passively send along fragments
+// rather than reassembling and then fragmenting them such that they can be
+// sent to the user's watch client.
+func WithFragments() OpOption { return func(op *Op) { op.combineFragments = true } }
```

Comment:
`WithFragments` should disable combining fragments and return the raw watch responses

Reply:
What would the use case be for having the client return raw watch responses to the user be? I was under the impression that `WithFragments` sort of an internal option to prevent the proxy's client form combining and then refragmenting before sending off.